test/jit/test_misc.py

```diff
@@ -345,3 +345,22 @@ def sum_f(x: BroadcastingList2[float]) -> float:
 
         self.assertTrue(torch.jit.script(sum_i)(4) == 8)
         self.assertTrue(torch.jit.script(sum_f)(4.5) == 9.)
+
+    def test_parse_ir_annotate(self):
+        ir = """
+        graph():
+          %3 : int[] = prim::Constant[value=annotate(List[int], [])]()
+          return (%3)
+        """
+        graph = torch._C.parse_ir(ir, True)
```

Comment:
nit, check result somehow ? maybe with 
```
%1 : bool = prim::isinstance[types=[int[]]](%3)
return (%1)
```

Reply:
hmm that didn't work because... it can't parse List[int] 😄 

I'm just checking `== []` right now, lmk if you have another suggestion on how to do this